package.json

```diff
@@ -33,8 +34,8 @@
     "docsify-cli": "^4.1.10",
     "execa": "^0.8.0",
     "jest": "^21.2.1",
-    "jest-cli": "^21.2.1",
```

Comment:
Heads up here @GantMan

Reply:
I don't think you're supposed to have jest and jest-cli

Reply:
Ahhh.  My brain read this as you removing jest. Oops.